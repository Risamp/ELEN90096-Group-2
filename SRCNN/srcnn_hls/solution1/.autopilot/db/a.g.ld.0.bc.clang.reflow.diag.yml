--- !Passed
Pass:            reflow-pragma-loop-unroll
Name:            PragmaPartiallyUnroll
DebugLoc:        { File: src/conv1.cpp, Line: 13, Column: 0 }
Function:        _Z5conv1PA255_A255_fPA1_A9_A9_fPfS1_
Args:            
  - String:          'Unrolling loop '''
  - LoopName:        VITIS_LOOP_28_4
  - String:          ''' ('
  - LoopLoc:         'src/conv1.cpp:28:23'
  - String:          ') '
  - String:          'in function '''
  - String:          conv1
  - String:          ''' partially '
  - String:          ''
  - String:          'with a factor of '
  - Factor:          '3'
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/conv1.cpp, Line: 13, Column: 0 }
Function:        _Z5conv1PA255_A255_fPA1_A9_A9_fPfS1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'clamp(int, int, int)'
    DebugLoc:        { File: src/util.cpp, Line: 82, Column: 0 }
  - String:          ''' into '''
  - Caller:          'conv1(float (*) [255][255], float (*) [1][9][9], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv1.cpp, Line: 13, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/conv2.cpp, Line: 12, Column: 0 }
Function:        _Z5conv2PA255_A255_fPA64_A1_A1_fPfS1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'clamp(int, int, int)'
    DebugLoc:        { File: src/util.cpp, Line: 82, Column: 0 }
  - String:          ''' into '''
  - Caller:          'conv2(float (*) [255][255], float (*) [64][1][1], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv2.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/conv3.cpp, Line: 12, Column: 0 }
Function:        _Z5conv3PA255_A255_fPA32_A5_A5_fPfS1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'clamp(int, int, int)'
    DebugLoc:        { File: src/util.cpp, Line: 82, Column: 0 }
  - String:          ''' into '''
  - Caller:          'conv3(float (*) [255][255], float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv3.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
Function:        _Z5srcnnPA255_A255_fPA1_A9_A9_fPfPA64_A1_A1_fS6_PA32_A5_A5_fS6_S1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'conv2(float (*) [255][255], float (*) [64][1][1], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv2.cpp, Line: 12, Column: 0 }
  - String:          ''' into '''
  - Caller:          'srcnn(float (*) [255][255], float (*) [1][9][9], float*, float (*) [64][1][1], float*, float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
Function:        _Z5srcnnPA255_A255_fPA1_A9_A9_fPfPA64_A1_A1_fS6_PA32_A5_A5_fS6_S1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'conv3(float (*) [255][255], float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv3.cpp, Line: 12, Column: 0 }
  - String:          ''' into '''
  - Caller:          'srcnn(float (*) [255][255], float (*) [1][9][9], float*, float (*) [64][1][1], float*, float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Missed
Pass:            reflow-burst-inference
Name:            CouldNotAnalyzeBoundsPattern
DebugLoc:        { File: src/conv1.cpp, Line: 28, Column: 23 }
Function:        _Z5conv1PA255_A255_fPA1_A9_A9_fPfS1_
Args:            
  - String:          Could not analyze the loop bounds
  - String:          ' _XLX_SEP_ '
  - LoopName:        VITIS_LOOP_28_4
  - String:          ' '
  - LoopLoc:         'src/conv1.cpp:28:23'
  - String:          ' '
  - Function:        'conv1(float (*) [255][255], float (*) [1][9][9], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv1.cpp, Line: 13, Column: 0 }
  - String:          ' '
...
