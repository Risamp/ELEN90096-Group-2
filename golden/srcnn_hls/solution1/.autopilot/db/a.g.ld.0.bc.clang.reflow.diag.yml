--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/conv1.cpp, Line: 13, Column: 0 }
Function:        _Z5conv1PA255_A255_fPA1_A9_A9_fPfS1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'clamp(int, int, int)'
    DebugLoc:        { File: src/util.cpp, Line: 82, Column: 0 }
  - String:          ''' into '''
  - Caller:          'conv1(float (*) [255][255], float (*) [1][9][9], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv1.cpp, Line: 13, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/conv2.cpp, Line: 12, Column: 0 }
Function:        _Z5conv2PA255_A255_fPA64_A1_A1_fPfS1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'clamp(int, int, int)'
    DebugLoc:        { File: src/util.cpp, Line: 82, Column: 0 }
  - String:          ''' into '''
  - Caller:          'conv2(float (*) [255][255], float (*) [64][1][1], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv2.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/conv3.cpp, Line: 12, Column: 0 }
Function:        _Z5conv3PA255_A255_fPA32_A5_A5_fPfS1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'clamp(int, int, int)'
    DebugLoc:        { File: src/util.cpp, Line: 82, Column: 0 }
  - String:          ''' into '''
  - Caller:          'conv3(float (*) [255][255], float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv3.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
Function:        _Z5srcnnPA255_A255_fPA1_A9_A9_fPfPA64_A1_A1_fS6_PA32_A5_A5_fS6_S1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'conv1(float (*) [255][255], float (*) [1][9][9], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv1.cpp, Line: 13, Column: 0 }
  - String:          ''' into '''
  - Caller:          'srcnn(float (*) [255][255], float (*) [1][9][9], float*, float (*) [64][1][1], float*, float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
Function:        _Z5srcnnPA255_A255_fPA1_A9_A9_fPfPA64_A1_A1_fS6_PA32_A5_A5_fS6_S1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'conv2(float (*) [255][255], float (*) [64][1][1], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv2.cpp, Line: 12, Column: 0 }
  - String:          ''' into '''
  - Caller:          'srcnn(float (*) [255][255], float (*) [1][9][9], float*, float (*) [64][1][1], float*, float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
--- !Passed
Pass:            reflow-inliner
Name:            AutoInline
DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
Function:        _Z5srcnnPA255_A255_fPA1_A9_A9_fPfPA64_A1_A1_fS6_PA32_A5_A5_fS6_S1_
Args:            
  - String:          'Inlining function '''
  - Callee:          'conv3(float (*) [255][255], float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/conv3.cpp, Line: 12, Column: 0 }
  - String:          ''' into '''
  - Caller:          'srcnn(float (*) [255][255], float (*) [1][9][9], float*, float (*) [64][1][1], float*, float (*) [32][5][5], float*, float (*) [255][255])'
    DebugLoc:        { File: src/srcnn.cpp, Line: 12, Column: 0 }
  - String:          ''''
...
